app/containers/wallet/dialogs/WalletTrezorConnectDialogContainer.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+
+import environment from '../../../environment';
+import type { InjectedDialogContainerProps } from '../../../types/injectedPropsType';
+
+import AboutDialog from '../../../components/wallet/trezorConnect/AboutDialog';
+import ConnectDialog from '../../../components/wallet/trezorConnect/ConnectDialog';
+import SaveDialog from '../../../components/wallet/trezorConnect/SaveDialog';
+
+import { ProgressStepOption } from '../../../stores/ada/TrezorConnectStore';
+import { Logger } from '../../../utils/logging';
+
+type Props = InjectedDialogContainerProps;
+@inject('stores', 'actions') @observer
+export default class WalletTrezorConnectDialogContainer extends Component<Props> {
+
+  static defaultProps = { actions: null, stores: null, children: null, onClose: () => {} };
```

Comment:
also remove for the same reason as the `@inject`

Reply:
DONE